common/config/rush/browser-approved-packages.json

```diff
@@ -230,6 +230,10 @@
       "name": "@itwin/projects-client",
       "allowedCategories": [ "common", "integration-testing", "internal" ]
     },
+    {
+      "name": "@itwin/reality-data-client",

```

Comment:
Since it is only used in test-app, it should probably have "integration-testing" for allowedCategories

Reply:
or be similar to @itwin/project-client

Reply:
made it similar to @itwin/project-client